holidays/countries/australia.py

```
@@ -118,6 +132,16 @@ def __init__(self, *args, **kwargs):
         kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
         super().__init__(*args, **kwargs)
 
+    def _move_holiday(
+        self,
+        dt: date,
+        *,
+        rule: ObservedRule | None = None,
+        force_observed: bool = False,
```

Ruff (0.15.15)

[warning] Unused method argument: `force_observed`

(ARG002)